### ISSUE TITLE
#168 broke colors on Windows

### ISSUE THREAD

**Issue by mrexodia** [MitMaro/git-interactive-rebase-tool#189]
It looks like #168 broke colors on Windows

1.1.0:

![image](https://user-images.githubusercontent.com/2458265/68688774-a66ea800-056f-11ea-9373-f505e1a82fe7.png)

93a0a37f52ce5807a527712ddb573eff92b68a01:

![image](https://user-images.githubusercontent.com/2458265/68688715-935bd800-056f-11ea-953b-9583c30cbcff.png)

**Comment by MitMaro** [comment]
Thanks for the report!

Hmm, I guess the color support functionality of pancurses doesn't work in Windows.

**Comment by MitMaro** [comment]
After much digging inside a Windows VM (I remember why I left Windows so many years ago), this ended up being caused by a casting bug.

`pancurses` exports a `chtype` for color pairs which is a `u64`/`u32` that is in *nix an integer value that starts from `0` and counts up based on the number of colors. Since I only registered 20 odd colors I decided to just cast this value to a `i16`.

Turns out the Windows does something else odd with the value that breaks when the value is cast to a `i16`.

The solution is to just use `chtype` directly without casting. I have the changes made locally and should have a fix up shortly.

As a side effect of this change, I discovered that Windows only supports 16 total colors, which is not enough to support selected lines. So that feature will have to be disabled in Windows.
